### PR TITLE
Improve the noise used by the volumetric lighting system (v2)

### PR DESCRIPTION
**Pull request by EvgeniiG**
### Purpose of this PR
- Improved the noise used by the volumetric lighting system.
- Removed the "slice distribution uniformity" parameter.

---
### Testing status

**Manual Tests**: Yes

**Yamato**: https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline/tree/HDRP%252Fcmj-noise/.yamato%252Fupm-ci-hdrp.yml%2523All_HDRP_trunk

---
### Comments to reviewers
Need to update docs.
Need to update the screenshots.

### CODE REVIEW

**Comment by iM0ve** [comment]
I don't know if this should be merged or not. There are no bugs. Based on this project fog seems very similar, a bit noisier near edges. It's hard for me to evaluate. Maybe Evgenii could provide some images with cases where the influence of the PR would be more pronounced? Or we could get some input from artists?
These are my test results:
https://confluence.unity3d.com/display/HDRP/Volumetric+lighting+system+V2

**Comment by EvgeniiG** [comment]
>Based on this project fog seems very similar, a bit noisier near edges.

I already commented on this.

image

**Comment by EvgeniiG** [comment]
Note that testing without filtering or reprojection enabled is pointless. That's not an intended scenario and is not directly correlated to the visuals with filtering or reprojection enabled.

**Comment by iM0ve** [comment]
Updated the confluence link with new captures to test the features when objects are in motion.
There is a bright light, one of the original lights and the camera is moving.

**Comment by iM0ve** [comment]
TL:DR Brings minor improvements, but dont recommend merging.

We did the last pass on this PR together with tech artist Sean Puller.
This PR indeed brings a minor improvement to the noise when reprojection is on and objects are moving. However when reprojection is off, the drop in noise quality is more noticeable (especially around edges as Evgenii explained). I think the improvement is too small to justify downgrade without reprojection. 

Reprojection ON all the time is not a viable option(ghosting) if we have moving lights (not directly related to this PR)
reprojection

No reprojection Old
Old_No_Reprojection

No reprojection PR (new)
New_No_Reprojection
